cmd/kubehound/config.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/DataDog/KubeHound/pkg/cmd"
+	"github.com/DataDog/KubeHound/pkg/telemetry/log"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
```

Comment:
I guess we have this dep already so it's fine, but otherwise I'd ask the reason of not being on v3.